test-files/cmd/glacial/RubberSheetConflateTest/output.osm

```diff
@@ -5453,21 +5453,20 @@
         <tag k="name" v="14th St SW"/>
         <tag k="error:circular" v="5"/>
     </way>
-    <way visible="true" id="-27503" timestamp="1970-01-01T00:00:00Z" version="1">
+    <way visible="true" id="-27495" timestamp="1970-01-01T00:00:00Z" version="1">
```

Comment:
This output got significantly better with several more ways joined up than before.